### ISSUE TITLE
Add brief instructions on how to generate QR code

### ISSUE THREAD

**Issue by saleel** [anon-aadhaar/anon-aadhaar#222]
Add brief instructions on how one can obtain QR code from the mAadhaar app. 

This should be an expandable section within the <SignInWithAadhaar> modal. A gif of steps recorded from the phone would be ideal.

**Comment by Vedant-asati** [comment]
Yes. Even if I generate a QR code from mAadhaar app, its showing invalid QR. It will be helpful if there is some guide for this.

**Comment by Divv5-jpg** [comment]
I’d like to work on this, assigning myself if possible!
